beep/tests/test_collate.py

```diff
@@ -55,3 +50,7 @@ def test_all_filenames(self):
                 Path(os.path.join("data-share", "raw_cycler_files", filename)).touch()
             process_files_json()
         pass  # to exit scratch dir context
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
Missing a newline at the end.

Reply:
Added. By the way, it's possible to set up a linter (e.g. `flake8`) to automatically check this kind of thing for you as part of the CI. There are also some automatic code factoring tools (I like `black` for python) which do most of the formatting work for you (add spaces before and after operators, add new lines at the end of files, etc)

Reply:
Thanks for the tip, will check it out!